code/game/objects/structures/beds_chairs/alien_nest.dm

```diff
@@ -32,9 +32,9 @@
 					span_hear("You hear squelching..."))
 			else
 				M.visible_message(span_warning("[M.name] struggles to break free from the gelatinous resin!"),\
-					span_notice("You struggle to break free from the gelatinous resin... (Stay still for two minutes.)"),\
+					span_notice("You struggle to break free from the gelatinous resin... (Stay still for one minute.)"),\
 					span_hear("You hear squelching..."))
-				if(!do_after(M, 1200, target = src))
+				if(!do_after(M, 60 SECONDS, target = src))
```

Comment:
I'm not sure how to feel about this, 60 seconds seems real quick, especially since the gestation would still be twice the old delay. I do like the whole incentivization of keeping people alive to pop tho, that vibes